sdks/java/harness/src/main/java/org/apache/beam/fn/harness/DoFnPTransformRunnerFactory.java

```diff
@@ -225,13 +256,13 @@ public final RunnerT createRunnerForPTransform(
         throw new IllegalArgumentException("Malformed ParDoPayload", exn);
       }
 
-      ImmutableListMultimap.Builder<TupleTag<?>, FnDataReceiver<WindowedValue<?>>>
```

Comment:
Thanks for removing the TupleTag usage. Unfortunately, it looks like we're still using it all over the place even in portable code. :/

Reply:
Yeah, its unfortunate but we'll need to convert to tuple tags in places still because a lot of the support code is used in both legacy and portable pipelines.